tests/integration/tables/keychain_items.cpp

```diff
@@ -14,34 +14,31 @@
 namespace osquery {
 namespace table_tests {
 
-class keychainItems : public testing::Test {
+class KeychainItemsTest : public testing::Test {
  protected:
   void SetUp() override {
     setUpEnvironment();
   }
 };
 
-TEST_F(keychainItems, test_sanity) {
-  // 1. Query data
+TEST_F(KeychainItemsTest, test_sanity) {
   auto const data = execute_query("select * from keychain_items");
-  // 2. Check size before validation
-  // ASSERT_GE(data.size(), 0ul);
-  // ASSERT_EQ(data.size(), 1ul);
-  // ASSERT_EQ(data.size(), 0ul);
-  // 3. Build validation map
-  // See helper.h for avaialbe flags
-  // Or use custom DataCheck object
-  // ValidationMap row_map = {
-  //      {"label", NormalType}
-  //      {"description", NormalType}
-  //      {"comment", NormalType}
-  //      {"created", NormalType}
-  //      {"modified", NormalType}
-  //      {"type", NormalType}
-  //      {"path", NormalType}
-  //}
-  // 4. Perform validation
-  // validate_rows(data, row_map);
+  ASSERT_GE(data.size(), 0ul);
```

Comment:
```suggestion
  ASSERT_GT(data.size(), 0ul);
```

I think you need to ensure there is a result here -- otherwise there's nothing to validate.

Reply:
Thanks for spotting this, I've updated the assert.